### ISSUE TITLE
how to use @StateConfig with ui-router nested views

### ISSUE THREAD

**Issue by PLEEROCK** [ngUpgraders/ng-forward#117]
Is there such ability? no info about it in the docs and examples

**Comment by tolemac** [comment]
Sample here https://github.com/brandonroberts/ng-forward-ui-router-example

**Comment by PLEEROCK** [comment]
@tolemac probably I wrote it wrong. I was talking about multiple root views at the same level with its subviews

**Comment by tolemac** [comment]
I think you have to do nothing new, Have you try this?

	@Component({
	  selector: 'app',
	  template: `
		<div ui-view="view1"></div>
		<div ui-view="view2"></div>
		`
	})
	@StateConfig([
	  { url: '/', views: {
			  'view1': {
				templateUrl: 'report-filters.html',
				controller: ComponentView1
			  },
			  'view2': {
				templateUrl: 'report-table.html',
				controller: ComponentView2
			  }
			}
        },
	])
	export class App {

	}

**Comment by timkindberg** [comment]
Yeah I should add a feature to route a component to a named view. Right now that's not supported I don't think.

**Comment by tolemac** [comment]
I think ng-forward should wait for ngComponentRouter, at the moment I wouldn't loss time with ui-router. 

IMHO.

**Comment by timkindberg** [comment]
Well ui-router has been the leading router. It will support Angular 2 at some point. Right now it's the best we got. We plan to support both it and the new router.

**Comment by ktersius** [comment]
So I looked around and I could not find any code samples defining abstract states without a url or component defined.

From what I could see if you try to define such states you get the following error : 
```
Uncaught TypeError: TypeError while analyzing StateConfig 'App' state components.
    Invalid Providers: please make sure all providers are an Injectable(), Component(), Directive(), a Provider, or a module string.
    Here's the invalid values: , 
```
